guides/common/modules/proc_reclaiming-postgresql-space.adoc

```diff
@@ -26,3 +26,18 @@ Use this procedure to reclaim some of this disk space on {Project}.
 ----
 # {foreman-maintain} service start
 ----
+. Confirm files are present under `/var/lib/pgsql/` and check its size.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# ls -l /var/lib/pgsql/
+
+# du -sh /var/lib/pgsq/
+----
+
+. If you are using {ProjectServer} {ProjectVersionPrevious} and higher, the data from `/var/lib/pgsql/` can be deleted.
```

Comment:
We only document for the specific version in the specific version of the docs. 
```suggestion
. Delete the data from the `/var/lib/pgsql/` directory: 
```

Reply:
@melcorr I made the requested changes and made another commit. Let me know if there are any issues seeing my commit because my VPN went down when I initially submitted the commit.

Reply:
Hey @bangelic can you do 

git log 
see can you see your second commit. 
Then do git push origin. It isn't here yet :-)